src/content/docs/ts-sdk/guides/transitions.mdx

```diff
@@ -41,9 +41,11 @@ function App() {
   }, []);
 
   return (
-    <View backgroundColor="#4d4d4d">
-      <Rescaler transition={{ durationMs: 2000 }}>
-        <InputStream inputId="input_1" />
+    <View style={{ backgroundColor: "#4d4d4d" }}>
+      <Rescaler
+        style={{ width: beforeTransition ? 480 : 1280 }}
+        transition={{ durationMs: 2000 }}>
+        <InputStream inputId="video" />
```

Comment:
I'm not entirely sure if that's what should be here, but based on the context and my testing this seems correct